### PR TITLE
isotonic pills

### PR DESCRIPTION
**Pull request by 567Turtle**
## About The Pull Request

Adds a few seasonal tips that show at the beginning of the round
AND
Isotonic pills which you can vend from the medic loadout vendor for 4 points to avoid having to go through the hassle of making them yourself for free

## Why It's Good For The Game

As a corpsman I can testify that having to make my own isotonic pills at the start of every round stinks, so you can now spend a whopping FOUR whole points to buy 16 pills which you can feed to your ungas to get their juice pack'd ass back in action

Also my OCD ridden monkey brain can't stand seeing that shitty custom pill bottle sprite with no initials, it's such a big contrast to the other pill bottles in the belt. This PR is mainly just to set aside the neuron activation I get from seeing that, even if I have to spend 4 points to get them, it'll be worth it to not have to look at the lightly fried pill bottle.

## Changelog
:cl:
add: Seasonal round tips
add: Isotonic pills
/:cl:

### CODE REVIEW

File: code/game/objects/items/reagent_containers/pill.dm [hunk 1]
```diff
@@ -158,6 +158,11 @@
 	list_reagents = list(/datum/reagent/medicine/tramadol = 15)
 	pill_id = 7
 
+/obj/item/reagent_containers/pill/isotonic
+	pill_desc = "A pill with an isotonic solution inside. Used to stimulate blood regeneration."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 10, /datum/reagent/iron = 10, /datum/reagent/consumable/sugar = 5)
```
Comment (QualityVan) [inline code/game/objects/items/reagent_containers/pill.dm:163]: Pills are 15u, as a rule. I'd either go 5/5/5 or just a 7.5 split on nutri/iron. Call it fortified nutrition or something, maybe.

Comment (567Turtle) [reply code/game/objects/items/reagent_containers/pill.dm:163]: didnt know about the pill cap, thanks

**Comment by QualityVan** [comment]
The tip additions should be in meme instead of meta, and the jim one already exists.

**Comment by Strayed-Raven** [comment]
Imagine spending points for this?

**Comment by 567Turtle** [comment]
> Imagine spending points for this?

I can, I do it every round, the deep fried unlabelled pill bottle in my belt makes my monkey brain angry, it also just saves me a bunch of time at roundstart

**Review by TiviPlus — CHANGES_REQUESTED** [review]
tips are completely unrelated and need to be atomized

**Comment by 567Turtle** [comment]
> tips are completely unrelated and need to be atomized

1. what does atomizing mean I have the brain of a paralotithical homeosapien
2. I know the tips are unrelated, I was just digging around in the code and thought I'd kill two stones with one PR

**Comment by lbnesquik** [comment]
Put the tips into a different PR

**Comment by 567Turtle** [comment]
atomized

**Review by TiviPlus — APPROVED** [review]

**Comment by TiviPlus** [comment]
In future please refrain from using emojis in your PRs

**Comment by 567Turtle** [comment]
> In future please refrain from using emojis in your PRs

1984

**Pull request merged**